alphagov/paas-cf
# [#158938714] Send logs to logit.io

What
----

We want to send the logs of the platform to logit.io rather than to our own logsearch stack.

In this PR we configure the syslog-forwarder to send the logs to the logit.io endpoint

We have decided NOT to use mutual TLS. See other PRs https://github.com/alphagov/paas-bootstrap/pull/163

How to review
-------------

 - Upload the logit credentials based on  https://github.com/alphagov/paas-credentials/pull/125 and  https://github.com/alphagov/paas-bootstrap/pull/175
 - Apply this branch and run one deployment
 - Your logs would appear on the logit.io `paas dev` stack.

Related PRs
------------

  - alphagov/paas-logsearch-for-cloudfoundry#7
  - https://github.com/alphagov/paas-credentials/pull/125
  - https://github.com/alphagov/paas-bootstrap/pull/175

Who can review
--------------

Anyone but @keymon or @jpluscplusm
